### PR TITLE
Add support for RELAX NG schema in xmllint

### PR DESCRIPTION
**Pull request by 813gan**

### CODE REVIEW

File: flycheck.el [hunk 1]
```diff
@@ -12516,13 +12516,21 @@ See URL `http://xmlstar.sourceforge.net/'."
   :safe #'flycheck-string-or-nil-p
   :package-version '(flycheck . "31"))
 
+(flycheck-def-option-var flycheck-xml-xmllint-relaxng-path nil xml-xmllint
+  "An RELAX NG schema to validate against."
+  :type '(choice (const :tag "None" nil)
+                 (file :tag "RELAX NG schema"))
+  :safe #'flycheck-string-or-nil-p
+  :package-version '(flycheck . "33"))
```
Comment (bbatsov) [inline flycheck.el:12524]: 33 is out now, so this should say 34.

**Comment by bbatsov** [comment]
The changes look good. Don't forget to reflect them in the changelog and I'll have them merged.

**Comment by 813gan** [comment]
> The changes look good. Don't forget to reflect them in the changelog and I'll have them merged.

I updated changelog. hopefully correctly.

File: flycheck.el (hunk 1 above)
Comment (813gan) [reply flycheck.el]: i pushed changed version

**Comment by bbatsov** [comment]
It will also be nice to mention the config option here https://www.flycheck.org/en/latest/languages.html#syntax-checker-xml-xmllint

**Comment by 813gan** [comment]
> It will also be nice to mention the config option here https://www.flycheck.org/en/latest/languages.html#syntax-checker-xml-xmllint

documentation and paperwork done

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**